docs/src/main/asciidoc/security-basic-auth-concept.adoc

```diff
@@ -0,0 +1,36 @@
+[id="security-basic-auth-concept"]
+= Basic Authentication
+ 
+HTTP Basic Authentication uses fields in the HTTP header and is the easiest to set up. Also, it is one of the least resource-demanding techniques that enforce access controls to the Web resources without requiring HTTP cookies, session identifiers, or login pages.
+
+In the context of an HTTP request, Basic authentication is a method for an HTTP user agent, such as a web browser, to provide a user name and password when creating a request. In Basic HTTP Authentication, a request contains a header field in the form of `Authorization: Basic <credentials>`, where credentials are the Base64 encoding of a user ID and password joined by a colon as described in the following example.
```

Comment:
```suggestion
An HTTP user agent, such as a web browser, uses an   `Authorization` header to provide a user name and password in each HTTP request.
The header is specified as `Authorization: Basic <credentials>`, where credentials are the Base64 encoding of the user ID and password joined by a colon, as shown in the following example.
```

One line per sentence. This could be differently combined with the previous grouping... ?

Reply:
Done